modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomLayouts.testcase

```diff
@@ -295,6 +700,107 @@ definition {
 			value1 = "Error:There can only be one default object layout");
 	}
 
+	@description = "LPS-165850 - Verify if you can 'select existing one' and add existing entries"
+	@priority = "5"
+	test CanSelectExistingEntriesOnRelationshipTab {
+		property portal.acceptance = "true";
+
+		task ("Given a related object with Custom Layout and Relationship tab") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 171628",
+				objectName = "CustomObject171628",
+				pluralLabelName = "Custom Objects 171628");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject171628");
+
+			ObjectAdmin.addObjectRelationshipViaAPI(
+				objectName_1 = "CustomObject171628",
+				objectName_2 = "CustomObject171628",
+				relationshipLabel = "Relationship",
+				relationshipName = "relationship",
+				relationshipType = "oneToMany");
+
+			ObjectAdmin.addObjectLayoutViaAPI(
+				layoutName = "Custom Layout",
+				objectName = "CustomObject171628");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject171628");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 171628");
+
+			CreateObject.selectTitleField(fieldLabel = "Custom Field");
+
+			CreateObject.saveObject();
+
+			ObjectAdmin.goToLayoutsTab();
+
+			ObjectAdmin.selectSpecificKebabMenuOption(
+				kebabOption = "View",
+				objectLabel = "Custom Layout");
+
+			ObjectAdmin.markLayoutAsDefault();
+
+			ObjectAdmin.goToLayoutTabOnLayouts();
+
+			ObjectAdmin.addTabFieldsOnLayout(
+				blockName = "Block 1",
+				columnsNumber = "1",
+				fieldList = "Custom Field",
+				tabName = "Field Tab");
+
+			ObjectAdmin.addTabRelationshipsOnLayout(
+				relationshipsLabelName = "Relationship",
+				tabName = "Relationship Tab");
+
+			PortletEntry.save();
+		}
+
+		task ("When add many entries") {
+			for (var letter : list "A,B,C,D,E,F,G,H,I,J,K") {
```

Comment:
Is there a reason this test needs this many entries? Can it be done with 3-5? If this amount is necessary, then we don't have to change it yet. Otherwise, having multiple entries that don't affect the outcome of the test is unnecessary.